files/grest/rpc/01_cached_tables/active_stake_cache.sql

```diff
@@ -162,10 +171,43 @@ $$
       ) DO UPDATE
         SET AMOUNT = EXCLUDED.AMOUNT;
 
-      PERFORM grest.update_control_table(
-        'last_active_stake_validated_epoch',
-        _epoch_no::text
-      );
+    /* ACCOUNT ACTIVE STAKE CACHE */
+    SELECT
+      COALESCE(MAX(epoch_no), 0)
+    FROM
+      GREST.ACCOUNT_ACTIVE_STAKE_CACHE
+    INTO _last_account_active_stake_cache_epoch_no;
+
+    INSERT INTO GREST.ACCOUNT_ACTIVE_STAKE_CACHE
+      SELECT
+        STAKE_ADDRESS.VIEW AS STAKE_ADDRESS,
+        POOL_HASH.VIEW AS POOL_ID,
+        EPOCH_STAKE.EPOCH_NO AS EPOCH_NO,
+        SUM(EPOCH_STAKE.AMOUNT) AS AMOUNT
+      FROM
+        EPOCH_STAKE
+        INNER JOIN POOL_HASH ON POOL_HASH.ID = EPOCH_STAKE.POOL_ID
+        INNER JOIN STAKE_ADDRESS ON STAKE_ADDRESS.ID = EPOCH_STAKE.ADDR_ID
```

Comment:
For consistency (tho we might wanna check if we missed to refer elsewhere), mentioning schema..

Reply:
> Need to add update to specs

yep, just wanted to check first if that's the endpoint we want and whether we want to use this cache in some existing endpoints.